currentColor: add definition and links (#24111)
mdn/content: files/en-us/web/css/color/index.md

- {{cssxref("&lt;color&gt;")}}
  - : Sets the color of the textual and decorative parts of the element.
- [`currentColor`](/en-US/docs/Web/CSS/color_value#currentcolor_keyword)

>   - : Normally sets the color to the element's `color` property value, but when set as the value of `color` it is treated as `inherit`.

Reviewer: ```suggestion
  - : Sets the color of the element's `color` property value. However, if set as the value of `color`, `currentColor` is treated as `inherit`.
```

should 'to' be 'of'?

Author: should be "to".

>   - : Sets the color to the element's `color` property value. However, if set as the value of `color`, `currentColor` is treated as `inherit`.

Author: ```suggestion
  - : Sets the color to the element's `color` property value. However, if set as the value of `color`, `currentcolor` is treated as `inherit`.
```
